pages/blog/migrating-from-alpha-to-beta.mdx

```diff
@@ -0,0 +1,306 @@
+```
+
+#### Theme configuration
+
+A couple of things here:
+
+- The `tokens` key in the configuration has been renamed to `theme`.
+- You no longer need prefix them with `$`.
+
+```diff
+createCss({
+- tokens: {
++ theme: {
+    colors: {
+-     $gray500: 'hsl(206,10%,76%)'
++     gray500: 'hsl(206,10%,76%)'
+    }
+  }
+});
+```
+
+#### Breakpoints configuration
+
+A couple of things here:
+
+- The `breakpoints` key in the configuration has been renamed to `conditions`.
+- A `condition` is now just a key-value pair. No more function.
+
+```diff
+createCss({
+- breakpoints: {
++ conditions: {
+-   bp1: (rule) => `@media (min-width: 640px) { ${rule} }`,
++   bp1: '@media (min-width: 640px)'
+  }
+});
+```
+
+#### Chaining psuedo classes and elements
+
+If you want to chain pseudo-classes or pseudo-elements you now must use the `&` qualififer.
+
+```diff
+{
+- ':hover': {}
++ '&:hover': {}
+- '::before': {}
++ '&::before': {}
+}
+```
+
+#### Responsive styles
+
+Responsive styles must now be wrapped in a `when` block.
+
+```diff
+{
+- bp1: {}
++ when: {
++   bp1: {}
++ }
+}
+```
+
+#### Referencing another Stitches component
+
+You now must prefix a Stitches component selector with a `.` (this is a dot).
+
+```diff
+const Icon = styled('svg', {});
+
+const Button = styled('button', {
+-  [`& ${Icon}`]: {
++  [`& .${Icon}`]: {
+    marginLeft: '6px',
+  },
+});
+```
+
+#### Compound variants
+
+This is now a special `compoundVariants` object in the style object.
+
+```diff
+const Button = styled('button', {
+  variants: {
+    color: {
+      violet: {},
+      gray: {},
+    },
+    appearance: {
+      outline: {}
+    }
+  },
++ compoundVariants: {
++   color: 'violet',
++   appearance: 'outline',
++   css: {
++     color: 'blueViolet',
++     borderColor: 'darkviolet',
++     '&:hover': {
++       color: 'white',
++     },
++   }
++ }
+})
+
+- Button.compoundVariant({
+-  color: 'violet',
+-  appearance: 'outline',
+- }, {
+-  color: 'blueViolet',
+-  borderColor: 'darkviolet',
+-  ':hover': {
+-    color: 'white',
+-  },
+- })
+```
+
+#### Server-Side Rendering
+
+A couple of things here:
+
+- The `getStyle` function has been deleted.
+- A `toString` function has been added.
+- The `toString` function receives no arguments, just call it and it'll render your styles as string.
+
+```diff
+- const { result, styles } = css.getStyles(() => renderSomething());
++ const styles = renderToString()
+```
+
+### New features
+
+We've also introduced some exciting new features, here's an overview.
+
+#### Reference tokens in the same scale
+
+You can now create semantic tokens by referencing tokens in the same scale, as always, use the `$` to reference a token.
+
+```diff
+createCss({
+  theme: {
+    colors: {
+      gray500: 'hsl(206,10%,76%)',
++     primary: '$gray500'
+    }
+  }
+})
+```
+
+#### Choose your theme class
+
+You can now define the class to be generated when creating additional themes.
+
+```diff
++ const darkTheme = ('dark-theme', {})
```

Comment:
It looks like something is missing here, right? (I don't know the new api, just curious in the beta)
I'm guessing a `theme` function call? :D
```suggestion
+ const darkTheme = theme('dark-theme', {})
```

Reply:
haha yeah 😂 glad its intuitive enough! haha